apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -0,0 +1,284 @@
+import React, {Component, PropTypes} from 'react';
+import color from "../../util/color";
+import {Table, sort} from 'reactabular';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+import PasswordReset from './PasswordReset';
+import ShowSecret from './ShowSecret';
+import i18n from "@cdo/locale";
+
+export const studentSectionDataPropType = PropTypes.shape({
+  id: PropTypes.number,
+  name: PropTypes.string,
+  username: PropTypes.string,
+  userType: PropTypes.string,
+  age: PropTypes.number,
+  gender: PropTypes.string,
+  secretWords: PropTypes.string,
+  secretPictureName: PropTypes.string,
+  secretPicturePath: PropTypes.string,
+  sectionId: PropTypes.number,
+});
```

Comment:
Are any/all of these PropTypes required?

Reply:
Added one, but will probably adjust as I start using this on pages with real data.